lib/sie_exporter.rb

```diff
@@ -0,0 +1,117 @@
+# encoding: utf-8
+
+require 'sie'
+
+# Exports activity years to SIE.
+class SieExporter
+  def initialize(activity_year)
+    @activity_year = activity_year
+  end
+
+  def call
+    Sie::Document.new(self).render
+  end
+
+  def program
+    'MAGE'
+  end
+
+  def program_version
+    `git rev-parse --short HEAD`.chomp
```

Comment:
This line depends on the working directory being tracked by git, which is not the case in production. Maybe add some kind of fallback here?